### PR TITLE
Simplify text in buttons on Join page

### PR DESCRIPTION
**Pull request by ewels**
As we get more buttons on the Join page, they're getting a bit wordy.

Now:

image

Before:

image

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1449"></a>

### CODE REVIEW

File: public_html/join.php [hunk 1]
```diff
@@ -24,20 +24,20 @@
 <p>We use a few different tools to organise the nf-core community -
     you are welcome to join us at any or all!</p>
 <p class="text-center">
-  <a href="https://nf-co.re/join/slack" class="mb-2 btn btn-lg btn-success">
-    <i class="fab fa-slack"></i> Join nf-core Slack
+  <a href="https://nf-co.re/join/slack" class="mb-2 me-3 btn btn-lg btn-success">
```
Comment (mashehu) [inline public_html/join.php:27]: because we use the brand colors for the rest:
```suggestion
  <a href="https://nf-co.re/join/slack" class="mb-2 me-3 btn btn-lg btn-success" style="background-color: #4a154b;">
```

**Review by mashehu — APPROVED** [review]

File: public_html/join.php (hunk 1 above)
Comment (ewels) [reply public_html/join.php:27]: Ok, but has a green border so needs to be btn-dark as well.
```suggestion
  <a href="https://nf-co.re/join/slack" class="mb-2 me-3 btn btn-lg btn-dark" style="background-color: #4a154b;">
```
